plugins/acpi-facp/fu-plugin-acpi-facp.c

```diff
@@ -41,6 +41,12 @@ fu_plugin_acpi_facp_add_security_attrs(FuPlugin *plugin, FuSecurityAttrs *attrs)
 		return;
 	}
 	if (!fu_acpi_facp_get_s2i(facp)) {
+		FwupdBiosAttr *bios_attr =
+		    fu_context_get_bios_attr(fu_plugin_get_context(plugin), "SleepState");
+		if (bios_attr != NULL) {
+			fwupd_security_attr_set_bios_attr_id(attr,
+							     fwupd_bios_attr_get_id(bios_attr));
+		}
```

Comment:
I think we do need to say what bios attr value we need to fix this. e.g. have something like `fwupd_security_attr_set_bios_value` making it clear that this value is the *desired* value, not the actual value.

Reply:
Right. I think we need two keys then. Like a desired value key and which BIOS setting to change key.

Reply:
Agree.